sources/packages/web/src/components/common/InstitutionUserSummary.vue

```diff
@@ -218,21 +215,28 @@ export default {
           enabled,
         );
         await getAllInstitutionUsers();
-        toast.success(
-          "User status updated",
-          `${userDetails.displayName} is ${enabled ? "enabled" : "disabled"}`,
+        emitter.emit(
+          "snackBar",
+          toast.success(
+            `${userDetails.displayName} is ${enabled ? "enabled" : "disabled"}`,
+          ),
         );
       } catch (error: unknown) {
         if (
           error instanceof ApiProcessError &&
           error.errorType === INSTITUTION_MUST_HAVE_AN_ADMIN
         ) {
-          toast.warn("Cannot disable the institution admin", error.message);
+          emitter.emit(
```

Comment:
As we talked in the call, lets move this to a composable and made it simple to call 😉

Reply:
done 👍